frontend/src/lib/components/ExpandIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import clsx from 'clsx'
+
+// Imitates how Antd renders the expand icon
+// https://github.com/ant-design/ant-design/blob/master/components/table/ExpandIcon.tsx
+
+interface ExpandIconProps {
+    prefixCls: string
+    onExpand: (record: any, e: React.MouseEvent<HTMLElement>) => void
+    record: any
+    expanded: boolean
+    expandable: boolean
+    children?: JSX.Element
+}
+
+function ExpandIcon({ prefixCls, onExpand, record, expanded, expandable, children }: ExpandIconProps): JSX.Element {
+    const iconPrefix = `${prefixCls}-row-expand-icon`
+    return (
+        <div
+            style={{ display: 'flex', alignItems: 'center' }}
+            onClick={(e) => {
+                onExpand(record, e!)
```

Comment:
Let's remove the `!`, it doesn't make a difference anyway. For added plus points, let's convert this eslint rule from `warn` to `error`. We've been eliminating all `!`s anyway, since they produce warnings that add noise in github... and these `!`s should be at least warnings I think :).